src/ocamlorg_web/lib/redirection.ml

```diff
@@ -459,12 +478,13 @@ let redirect_p pattern =
 let fwd_v2 origin =
   Dream.get origin (fun req -> Dream.redirect req (Url.v2 ^ origin))
 
+let serve url path = Dream.get url (Dream.static path)
+
 let manual =
   [
+    serve "/manual/**" "/manual/";
```

Comment:
I don't think we should be configuring pages to be served from the `redirection.ml` module. Instead, this should be put into `router.ml`.

Reply:
I have moved this to `router.ml`